app/views/catalog/_index_location.html.erb

```diff
@@ -1,5 +1,7 @@
 <% if document.holdings.present? %>
-  <% document.holdings.libraries.select(&:present?).each do |library| %>
-    <%= render SearchResult::LocationComponent.new(library:, document:) %>
-  <% end %>
+  <div data-controller="live-lookup" data-live-lookup-url-value="<%= availability_index_path %>">
```

Comment:
Can we move this up to the existing div in `_accordion_section_library.html.erb`?

Reply:
Are you talking about that line or the entire template? 
I wouldn't move that line outside that if statement. We could move the line plus the if statement to _acccordian_section_library. I don't think that would cause any issues.

In terms of the template I think that is a lot of refactoring and moving things around which I think should probably be a separate PR.

Reply:
Moving it outside of the component means you've already circumvented the `render?` statement, so perhaps it needs to be in there.

Reply:
After having another look it seems like the javascript is querying for `[data-live-lookup-id]`, so it should just submit an empty query if there are no holdings present.  I think this div (just this line) can move up to `_accordion_section_library.html.erb`

Reply:
I am not sure why we would want to run JS if we don't have to.

Reply:
I'm pretty sure that if `_accordion_section_library` is rendering anything, then it's going to check availability.

Reply:
good point. I can't think of any record that would have not have document.holdings if it is being rendered by accordion_section_library and it doesn't look like index_location is being used by anything else.